stea/stea_input.py

```diff
@@ -67,7 +67,9 @@ def __init__(self, argv):
             self.config = config
 
         except Exception as ex:
-            raise ValueError(f"Could not load config file: {args.config_file}") from ex
+            raise ValueError(
+                f"Could not load config file: {args.config_file}, error: {ex}"
```

Comment:
This is not a pretty change, but allows `pytest.raises(match="something")` to work on the inner exception.